task/sinker.go

```diff
@@ -111,15 +111,17 @@ func (s *Sinker) Run() {
 	}
 
 	if s.rcm == nil {
-		if _, err = os.Stat(s.cmdOps.LocalCfgFile); err == nil {
-			if newCfg, err = config.ParseLocalCfgFile(s.cmdOps.LocalCfgFile); err != nil {
-				util.Logger.Fatal("config.ParseLocalCfgFile failed", zap.Error(err))
-				return
-			}
-		} else {
+		if _, err := os.Stat(s.cmdOps.LocalCfgFile); err != nil {
 			util.Logger.Fatal("expect --local-cfg-file or --nacos-dataid")
 			return
 		}
+
+		if newCfg, err = config.ParseLocalCfgFile(s.cmdOps.LocalCfgFile); err != nil {
+			util.Logger.Fatal("config.ParseLocalCfgFile failed", zap.Error(err))
+			return
+		}
+		applyCredentials(newCfg, s.cmdOps.Credentials)
```

Comment:
Move  `applyCredentials` invoking into `Config.Normallize()` since rcm(currently Nacos only) also need these credentials.

Reply:
Fixed this one